.dockerignore

```diff
@@ -2,4 +2,5 @@ bin
 client
 dist
 docs
+backend
 server
```

Comment:
Can we remove this?

Reply:
The file? or the line?

Reply:
the line.  not sure why the file specifies a directory that no longer exists....